tools/prlint/README.md

```diff
@@ -27,9 +27,12 @@ npm install
 
 # Usage
 
-```console
-node tools/prlint/index.js mandatoryChanges
+```javascript
+const linter = require('prlint')
+await linter.validatePr(8892);
+```
```

Comment:
Maybe specify this as a command using `node -e` so it can still be run from the console.

Or maybe write a tiny wrapper script that takes the PR# as a CLI arg?